Description: Hardcode python3 path
 There is no need to use /usr/bin/env, as we know what the path is on Debian.
 
 Also, we want to force the use of python3 even while the default is python2.
Forwarded: not-needed
Origin: vendor
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2019-01-17
--- a/attic/calc_tickadj/calc_tickadj
+++ b/attic/calc_tickadj/calc_tickadj
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 #
 # drift of 104.8576 -> +1 tick.  Base  of 10000 ticks.
 #
--- a/contrib/cpu-temp-log
+++ b/contrib/cpu-temp-log
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # coding: utf-8
 """\
 Usage: cpu-temper-log
--- a/contrib/smartctl-temp-log
+++ b/contrib/smartctl-temp-log
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # coding: utf-8
 """\
 Usage: smartctl-temp-log [device]
--- a/contrib/temper-temp-log
+++ b/contrib/temper-temp-log
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # coding: utf-8
 """\
 Usage: temper-temper-log
--- a/wafhelpers/pythonize-header
+++ b/wafhelpers/pythonize-header
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 """
 Generate a Python module from a C file of macro definitions.
--- a/devel/linkcheck
+++ b/devel/linkcheck
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 #
 # linkcheck - check link integrity in an asciidoc document tree
 #
--- a/contrib/zone-temp-log
+++ b/contrib/zone-temp-log
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # coding: utf-8
 """\
 Usage: zone-temper-log
--- a/contrib/make-leap-seconds.py
+++ b/contrib/make-leap-seconds.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """\
 make-leap-seconds.py - make leap second file for testing
 
--- a/contrib/ntpconfigtest
+++ b/contrib/ntpconfigtest
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 # On Linux systems, read /proc/config.gz and check for the
 # needed kernel options.
--- a/contrib/ntpheat
+++ b/contrib/ntpheat
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 #
 # generate some heat!
 #
--- a/contrib/ntpheatusb
+++ b/contrib/ntpheatusb
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 #
 # generate some heat!
 #
--- a/devel/python_paths.py
+++ b/devel/python_paths.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 """Find all Pythons in the command path, and corresponding exe/lib locations.
 
--- a/tests/pylib/jigs.py
+++ b/tests/pylib/jigs.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function, division
--- a/tests/pylib/test_packet.py
+++ b/tests/pylib/test_packet.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function, division
--- a/tests/pylib/test_statfiles.py
+++ b/tests/pylib/test_statfiles.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 import unittest
--- a/tests/pylib/test_util.py
+++ b/tests/pylib/test_util.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function
--- a/ntpclients/ntpdig.py
+++ b/ntpclients/ntpdig.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 """
 ntpdig - simple SNTP client
--- a/ntpclients/ntpkeygen.py
+++ b/ntpclients/ntpkeygen.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 '''
 ntpkeygen - generate cryptographic keys for NTP clients and servers
--- a/ntpclients/ntploggps.py
+++ b/ntpclients/ntploggps.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 """\
--- a/ntpclients/ntplogtemp.py
+++ b/ntpclients/ntplogtemp.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 """\
--- a/ntpclients/ntpmon.py
+++ b/ntpclients/ntpmon.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 # SPDX-License-Identifier: BSD-2-clause
--- a/ntpclients/ntpq.py
+++ b/ntpclients/ntpq.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 #
 # ntpq - query an NTP server using mode 6 commands
--- a/ntpclients/ntpsnmpd.py
+++ b/ntpclients/ntpsnmpd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function, division
--- a/ntpclients/ntpsweep.py
+++ b/ntpclients/ntpsweep.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 """
 ntpsweep - print various information about given NTP servers
--- a/ntpclients/ntptrace.py
+++ b/ntpclients/ntptrace.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 """
 ntptrace - trace peers of an NTP server
--- a/ntpclients/ntpviz.py
+++ b/ntpclients/ntpviz.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 """\
 ntpviz - visualizer for NTP log files
--- a/ntpclients/ntpwait.py
+++ b/ntpclients/ntpwait.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 """\
 ntpwait - Wait for ntpd to stabilize the system clock.
--- a/pylib/agentx.py
+++ b/pylib/agentx.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function, division
--- a/tests/pylib/test_agentx_packet.py
+++ b/tests/pylib/test_agentx_packet.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 import unittest
